src/Tasks/Microsoft.NET.Build.Tasks/GenerateRuntimeConfigurationFiles.cs

```diff
@@ -138,7 +141,7 @@ protected override void ExecuteCore()
                     projectContext.IsFrameworkDependent,
                     projectContext.LockFile.PackageFolders);
 
-                if (writeDevRuntimeConfig)
+                if (GenerateRuntimeConfigDevFile)
                 {
                     WriteDevRuntimeConfig(projectContext.LockFile.PackageFolders);
```

Comment:
Consider checking that `RuntimeConfigDevPath` is not empty - the code will throw an exception if that's the case currently.

Reply:
fixed, thanks!